thetis/equation.py

```diff
@@ -62,6 +65,10 @@ class Equation(object):
     SUPPORTED_LABELS = ['source', 'explicit', 'implicit', 'nonlinear']
 
     def __init__(self, function_space):
+        """
+        :arg function_space: :class:`FunctionSpace` where the solution belongs
```

Comment:
sorry, nonsense, "the FunctionSpace the solution belongs to" (or "the FunctionSpace where the solution belongs")